### ISSUE TITLE
/manage looks bad with omnibar added

### ISSUE THREAD

**Issue by themightychris** [SlateFoundation/slate-admin#20]
The site CSS pulled in for the omnibar makes SlateAdmin look bad AND the SlateAdmin CSS seems to be making omnibar look bad too.

I saw that there was a standalone omnibar.css file that maybe should be included instead of main.css here. That would be ideal, but in the long run we need the Slate frontend design and slate-theme to get along on the same pages like the old slate-theme did for CBL UIs

**Comment by themightychris** [comment]
@ryon the source of the omnibar's woes seems to be the x-body class getting added to the <body> tag. Our setting of `Ext.scopeCss = true` is supposed to prevent the class from being added to the body tag automatically on every page, but because we're using the Viewport class here it gets added anyway.

It seems our options are:
1. Stop using the Viewport class and render/size our own div that takes up the whole screen but the omnibar and render into that
2. make the omnibar look good even inside x-body
3. enhance slate-theme so that x-body on <body> is ok and still implements our desired frontend styling. Rules for making things smaller and ext-y get targeted at specific components, maybe with a mixin?

I think 3 is our best option, we already encountered this need (and mostly fixed it?) with the old slate-theme when I turned the CBL views into ext components.
